agent/command/s3_get.go

```diff
@@ -83,6 +84,12 @@ type s3get struct {
 	// that were generated by an ec2.assume_role command.
 	assumedRoleARN string
 
+	// existingCredentials holds the existing passed in temporary credentials
+	// and their expiration time. This is used to reuse passed in credentials
+	// whenever possible. User's don't pass in their expiration time, so we
+	// need to parse it from the session token.
```

Comment:
Is it parsed from the session token? It looks like it's just using the expiration time from the assumed role cache.